### DISCUSSION TITLE
Emulate Skylanders Portals within Dolphin

### DISCUSSION THREAD

**Discussion by Medard22** [Medard22/Dolphin-MMJR2-VBI#63] [Ideas]
Let´s continue our discussion here, shall we?

**Comment by Medard22** [comment]
I am slowly updating MMJR2 codebase and I am very close to the commit you mentioned. Adding support to the core should be pretty straightforward. To actually expose it in the Android UI is a different thing altogether. MMJR2 uses old Android UI. Dolphin redesigned most of it´s UI and moved to Kotlin. I really can´t promise you anything, but I will look into it.

**Comment by Medard22** [comment]
I have sucesfully ported over Skylanders support, however Android UI proves to be a real challenge. Dolphin moved away from old Android UI of MMJR2 . While I managed to port the official Dolphin UI implementation and fixed the most obvious issues. I still get a crash when I choose Skylander Portal settings in the menu. It should open a dialog window, when you can set your Skylanders. I am looking into it.

**Reply by panjidharma** [reply]
Nice brother i am still waiting for next update

**Reply by Kirbythepink1980** [reply]
Thank you in advance brother for your contributions to the community, it helps a lot of people especially those who use of mali gpu.

**Comment by Medard22** [comment]
Skylanders Portal support is not going to happen for now. I tried to write the Android UI part myself, but I failed miserably and eventually burned out. So I decided to skip it for now. If there is any Android developer willing to help me with this one I´m all ears.
